brief skeleton for events page

mjhajharia opened the pull request:
check #57 

This mostly resembles the organizers page structure and only has preliminary structure

Please drop any ideas about layout changes:

- Are we ok with this or do we prefer a table instead, like [this](https://pymc-devs.github.io/pymcon/schedule)
- Are only css changes needed for size/design of cards
- Should we use hugo [taxonomies](https://jpanther.github.io/congo/docs/content-examples/#taxonomy-pages) for authors / type of event etc.
- How much information about an event do we want on

  1. Event page
  2. Overall - as in do we want pages for authors, categories or other such things.

  Or do we just prefer simple tabulation - in a .md table or card whichever form

cc: @canyon289 @purna135 @cluhmann

purna135 commented:
Can you share a preview/screensort of the current changes?


Maybe we can add the card we made for the website's initial edition. which can be found here, https://purna135.github.io/pymcon-hugo/

![image](https://user-images.githubusercontent.com/42216008/208266715-5aba5f27-269d-47fa-8bb0-f40c8b0a5304.png)

canyon289 commented:
> Can you share a preview/screensort of the current changes?
> 
> Maybe we can add the card we made for the website's initial edition. which can be found here, https://purna135.github.io/pymcon-hugo/
> 
> ![image](https://user-images.githubusercontent.com/42216008/208266715-5aba5f27-269d-47fa-8bb0-f40c8b0a5304.png)

Using this card is an amazing idea! It has all the elements we need

canyon289 commented on layouts/_default/events.html:
```diff
@@ -0,0 +1,50 @@
+{{ define "customCSS" }}
```
Thanks for makign this easily extensible

canyon289 commented:
This is a good start. Thanks for adding the page and a configurable toml

Two questions
* Can we blend what you have with what @purna135 has on his card? Perhaps it'll be possible to copy paste that code so it isn't too much work to recreate?
* Is it possible to have an  a featured event on the front page?  Perhaps it can be a flag on each event configuration file  if its featured or not in toml and if it is it shows up on the front page We can do this in another PR if need be.

![image](https://user-images.githubusercontent.com/7213793/208268927-4f9ab8a9-6c81-480a-adea-2dfc9838e449.png)

canyon289 commented:
Bigger screenshot for reference 

![image](https://user-images.githubusercontent.com/7213793/208269011-37468b7a-2c91-4805-a6f6-2689d67a9788.png)

purna135 commented:
> Using this card is an amazing idea! It has all the elements we need. Curious how it matches up to the card in this PR.


We already have this card; I recall commenting these lines of code when we made the website public.

See PR, https://github.com/pymc-devs/pymcon_web_series/pull/152

I believe we can enable it and create a separate page for events.

purna135 commented:
> Can we blend what you have with what @purna135 has on his card?

If you prefer, I can uncomment the code for this card and create a separate page for the event. Because I already designed this card, it will take me less time.

reshamas commented:
@purna135 Do you think we can choose a size for the card that is consistent with the Meetup size?
Such as 1200 by 630 px?

purna135 commented:
Are you referring to this card? Yes, we can absolutely choose this size.

![image](https://user-images.githubusercontent.com/42216008/208313274-d358b242-d9da-4721-934f-d0dad0b0819d.png)

reshamas commented:
@purna135 Yes, this one, which shows the social card.

<img width="499" alt="Screen Shot 2022-12-19 at 12 40 47 PM" src="https://user-images.githubusercontent.com/2507232/208486861-e6572405-438d-4a3d-8fd9-3a144f8f8662.png">

purna135 commented:
Hi @mjhajharia, will you apply the changes mentioned above to this PR?

Or should I create a new PR for this?

cluhmann commented:
Is there anything we can do to help this along?  We have the rough event schedule confirmed, so we can begin filling in the skeleton with actual content relatively quickly after this is merged.

purna135 commented:
Let me open another PR that incorporates the above-mentioned changes.

mjhajharia commented:
hi! sorry for the long silence, I wasn't sure what was required because the `.toml` I added had all the attributes in the `incoming_talks` file and a few more. Anyway, I incorporated that card into the website as well. So now, you can simply add `true` or `false` to `highlighted` as an atttribute in `events.toml` to mark an event (for viewing it on the home page). Otherwise, it stays the same on the events page.


Here are (zoomed out) previews for the home page (featured events) as well as the events page.

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/55153708/211116462-9c1e0408-fe19-4314-9ad4-92ffadec30e3.png">

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/55153708/211116409-b1d703f7-ab44-4e42-b302-5eccc9cc76b6.png">

Let me know if this works!

canyon289 commented:
Other event page is merged so closing this one. Thanks @mjhajharia for your contributions as well
